app/views/shared/_new.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="new-costume" style="padding-top: 100px">
+  <%= form_for(costume) do |f| %>
+    <%= f.label :name %>
+    <%= f.text_field :name %>
+    <%= f.label :description %>
+    <%= f.text_field :description %>
+    <%= f.label :animal_type %>
+    <%= f.text_field :animal_type %>
+    <%= f.label :longitude %>
+    <%= f.number_field :longitude, input_html: {:value => 51.50} %>
+    <%= f.label :latitude %>
+    <%= f.number_field :latitude, input_html: {:value => 0} %>
+      <%= f.label :user_id %>
```

Comment:
I would omit these from the form - user doesn't need to know, nor should know, their id. 
if you need to pass a user id to costume instance, do so in the controller using current_user